page/blank.njk

```diff
@@ -62,6 +62,6 @@
         <script nonce="{{ cspNonce }}" src="/govuk-frontend/all.js"></script>
         <script nonce="{{ cspNonce }}">window.GOVUKFrontend.initAll()</script>
         <script nonce="{{ cspNonce }}" src="/dist/js/autocomplete.min.js"></script>
-        <script nonce="{{ cspNonce }}" src="/dist/js/bundle.js"></script>
+        <script nonce="{{ cspNonce }}" src="/dist/js/bundle.43fc86e32ee1d066333a.js?{{CW_APP_VERSION}}"></script>
```

Comment:
If I am understanding things correctly, `43fc86e32ee1d066333a` needs to be removed from this filename.